packages/backend/src/tests/app.ts

```diff
@@ -31,6 +31,7 @@ export const createTestApp = async (
   container: IocContract<AppServices>
 ): Promise<TestContainer> => {
   const config = await container.use('config')
+  config.env = 'test'
```

Comment:
Are there cases where it wouldn't already be set to this via `NODE_ENV`?

Reply:
Removed; I'm not sure why I added that since it appears to be unnecessary.